src/client/dfuse/dfuse_main.c

```diff
@@ -263,6 +263,8 @@ main(int argc, char **argv)
 
 	fuse_session_destroy(dfuse_info->di_session);
 
+	D_GOTO(out_dfs, 0);
```

Comment:
maybe add a comment here why we don't close dfs/cont/pool here and point to where it's being done.

Reply:
We can land this if it fixes an issue for you but I was unable to reproduce it locally, and the reference counting change that went in yesterday might change things as well.

I'm about to re-work the pool/container connect code anyway to allow for unified namespace so even if this lands I don't see it remaining in place for more than a few weeks.